src/platform/intel/cavs/platform.c

```diff
@@ -74,7 +74,11 @@ static const struct sof_ipc_fw_ready ready
 		.abi_version = SOF_ABI_VERSION,
 		.src_hash = SOF_SRC_HASH,
 	},
+#if CAVS_VERSION >= CAVS_VERSION_1_8
+	.flags = DEBUG_SET_FW_READY_FLAGS | SOF_IPC_INFO_D3_PERSISTENT,
```

Comment:
Btw, we should probably have the same prefixes for these boot info flags. Where are they defined ?